src/neural-network.js

```diff
@@ -35,12 +35,27 @@ export default class NeuralNetwork {
     };
   }
 
+  static get trainingValidations() {
```

Comment:
Do we need a `get` for this? Why not just define these in the validation method itself?

Reply:
sorry I didn't see your comments but yeah I got rid of the getter/setter.  Originally when they set this to false I was printing out a message showing what it would do, but I thought that overkill.